src/main/resources/META-INF/create.sql

```diff
@@ -32,8 +32,12 @@ alter table SECURITY_GROUP_INTERFACE_POLICY add constraint if not exists FK_SGI_
                     
 alter table SECURITY_GROUP_INTERFACE_POLICY add constraint if not exists FK_SGI_POLICY_POLICY foreign key (policy_fk) references POLICY;
 
-MERGE INTO Domain VALUES (1, 'Default');
+alter table domain MODIFY column id BIGINT auto_increment;
```

Comment:
can we do this a part of create table

Reply:
Yes, we can. But, it will break the existing use of plugin.